lib/core/test/data/Cardano/Wallet/Api/ApiByronWallet.json

```diff
@@ -1,330 +1,296 @@
 {
-    "seed": 8278408552904127992,
+    "seed": 757099553700796479,
     "samples": [
         {
             "passphrase": {
-                "last_updated_at": "1876-09-15T12:52:19.652881236802Z"
+                "last_updated_at": "1897-02-11T08:32:45Z"
             },
             "state": {
-                "status": "ready"
+                "status": "syncing",
+                "progress": {
+                    "quantity": 53,
+                    "unit": "percent"
+                }
             },
             "balance": {
-                "reward": {
-                    "quantity": 18,
-                    "unit": "lovelace"
-                },
```

Comment:
Not sure how this slipped through but we had ByronWallet with reward balance in the golden files :thinking: